index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
```

Comment:
In the time being this is still needed.

The GCM endpoint expects the authorization header to be the GCM Server API Key rather the the VAPID Bearer header.

GCM will continue to remain a thing for a few reasons: VAPID is only supported in Chrome Beta, but it's also the endpoint returned for Opera and Samsung Browser. I'm getting in touch with peeps at Opera and Samsung to see what the plan is, but it seems for the short term we should keep this in.